# DP: boehm-gc: use mmap instead of brk also on kfreebsd-*.

--- a/src/boehm-gc/configure.host
+++ b/src/boehm-gc/configure.host
@@ -41,7 +41,7 @@
 fi

 case "${host}" in
-  *-linux*)
+  *-linux*|*-kfreebsd*-gnu*)
     gc_use_mmap=yes
     ;;
 esac
